src/test/getarg_tests.cpp

```diff
@@ -30,7 +30,7 @@ static void ResetArgs(const std::string& strArg)
         vecChar.push_back(s.c_str());
 
     std::string error;
-    BOOST_CHECK(gArgs.ParseParameters(vecChar.size(), vecChar.data(), error));
+    BOOST_CHECK(argsman.ParseParameters(vecChar.size(), vecChar.data(), error));
```

Comment:
In commit "scripted-diff: replace gArgs with a local args manager" (af069fcdd9b5e2c983ee32f0292c8e1f167359ec)

In the commit message for https://github.com/bitcoin/bitcoin/commit/af069fcdd9b5e2c983ee32f0292c8e1f167359ec.patch you need an extra newline between the subject and body of the commit for the scripted diff to work like:

```
scripted-diff: replace gArgs with a local args manager

-BEGIN VERIFY SCRIPT-
sed -i -e '205!s/gArgs/argsman/g' src/test/getarg_tests.cpp
-END VERIFY SCRIPT-
```

Reply:
Thank you! I will update the developer README too then.